src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/data/CreateVisitRequest.kt

```diff
@@ -53,8 +44,12 @@ data class CreateVisitRequest(
 //  @Schema(description = "Staff who created the visit booking") //, required = true)
 //  // @NotNull
 //  val staffId: Long? = null,
-//
+
   @Schema(description = "Issue date", required = true)
   @NotNull
   val issueDate: LocalDate? = null,
+
```

Comment:
would time ever be useful here - events out of order etc

Reply:
Thing is, this maps to a Nomis DATE so we cant really 🤷